static/theme-flex/style.css

```
@@ -407,6 +407,8 @@ article > aside .menu .dd-item {
   list-style: none;
   font-size: 1rem;
 }
+article > aside .menu .dd-item.parent > a {
+  font-size: 18px; }
```

```suggestion
  display: inline-block;
  padding-top: 4px;
  font-size: 18px;
  }
```
This is kind of nit-picky, but I just think it needs a little extra spacing on the top, if you're open to adding that.